frontend/catalyst/utils/jax_cpu_lapack_kernels/lapack_kernels.cc

```diff
@@ -0,0 +1,914 @@
+/* Copyright 2021 The JAX Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+/*
+ * This file is a modified version of
```

Comment:
What was modified in this file?

Reply:
I've added a more detailed description of what was modified. Let me know if anything is unclear.